Source/SwiftLintFramework/Rules/ExplicitACLInTopLevelDeclarationRule.swift

```diff
@@ -0,0 +1,80 @@
+//
+//  ExplicitACLInTopLevelDeclarationRule.swift
+//  SwiftLint
+//
+//  Created by Jose Cheyo Jimenez on 4/28/17.
+//  Copyright © 2017 Realm. All rights reserved.
+//
+
+import Foundation
+import SourceKittenFramework
+
+public struct ExplicitACLInTopLevelDeclarationRule: OptInRule, ConfigurationProviderRule {
+    public var configuration = SeverityConfiguration(.warning)
+
+    public init() {}
+
+    public static let description = RuleDescription(
+        identifier: "explicit_acl_in_top_level_declaration",
+        name: "Explicit ACL in Top Level Declaration",
+        description: "Top-level declarations should specify Access Control Labels explicitly.",
```

Comment:
"ACL" stands for "Access Control Level", so perhaps reword to "Top-level declarations should specify Access Control Level keywords explicitly."